dashboard/test/factories/plc.rb

```diff
@@ -0,0 +1,56 @@
+FactoryGirl.allow_class_lookup = false
```

Comment:
I'd suggest moving PLC into PD. We should have never really made the distinction between the two, and this file is relatively small.